bin/CodeChecker.py

```diff
@@ -17,12 +17,12 @@
 
 import shared
 
-from cmdline_client import cmd_line_client
-from codechecker_lib import arg_handler
-from codechecker_lib import util
-from codechecker_lib import logger
-from codechecker_lib.logger import LoggerFactory
-from codechecker_lib.analyzers import analyzer_types
+from lib.cmd import cmd_line_client
+from lib import arg_handler
```

Comment:
I would prefer some more descriptive name like `libcodechecker`, I know this is not that short but still.